### ISSUE TITLE
Pasted arguments are not pre-checked

### ISSUE THREAD

**Issue by samaursa** [MBulli/SmartCommandlineArgs#112]
We have a _lot_ of CLI arguments. The copy/paste functionality to copy over the arguments works great, but sometimes you need a starting point with some defaults pre-checked, but that doesn't get copied over. I also cannot see in the `.json` file where the information to check/uncheck certain arguments is stored.

It's possible I am doing something wrong or maybe the feature has not been implemented yet?

**Comment by Irame** [comment]
Hi, I'm not quite sure what what you mean exactly. Do you mean that the checked state isn't copied over with copy'n'paste, or do you mean that the checked state isn't saved in the `.json` file and therefore all arguments are unchecked if you open se project for the first time.

The checked state should already be copied via copy'n'paste but if you mean the second thing, then I agree. It would be a good thing to store a default checked state which is used when the project is opened for the first time.

**Comment by samaursa** [comment]
I mean both actually. I tried copy-pasting just now and the checked state is not copied over (i.e. all the args are unchecked in the 2nd project that is open).

**Comment by MBulli** [comment]
Lets focus on the copy paste problem first. That's weird because that's what we tried but it worked for us. Do you copy the data between two VS instances or in the same window? Do you change solution between copy paste in the same VS instance?

btw the checked state is stored in the suo file because we classified that as user specific and not something you want to see in your source diffs. But as Irame suggested we might rework that.

**Comment by samaursa** [comment]
Sorry for the delayed reply. I just got back to work and am now able to test the args copying.

I do indeed change solutions between copy/paste. I tried again, but this time it worked and I realised what I was doing wrong. My fault, I will explain below.

I copied and then pasted the args in my notes. It seems like they lose extra information about the checkmarks since there is nothing in the pasted args that would suggest the arg is checkmarked or not.

Is there a way for me to make a backup of the args so that I _don't_ end up losing the checkmarks?
